open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-server/pom.xml

```diff
@@ -24,6 +24,12 @@
 
     <dependencies>
 
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>ffdc-services</artifactId>
+            <version>1.5-SNAPSHOT</version>
```

Comment:
This should be removed. The version is picked up from dependencyManagement in the top level pom